plugins/sqs/sqs.cpp

```diff
@@ -104,37 +104,37 @@ SQSHPCCPlugin::SQSHPCC::~SQSHPCC()
 }
 
 /**
- *  
+ *
  *
  *
  **/
 
-SQSHPCCPlugin::Response  SQSHPCCPlugin::SQSHPCC::sendMessage(const char* message) 
+SQSHPCCPlugin::Response  SQSHPCCPlugin::SQSHPCC::sendMessage(const char* message)
 {
 
   SQSHPCCPlugin::Response ref = {};
 
-  cout << "SendMessage is " << this->queueUrl << endl; 
+  cout << "SendMessage is " << this->queueUrl << endl;
 
   try {
     Aws::SQS::Model::SendMessageRequest sendMessageRequest;
     sendMessageRequest.SetQueueUrl(this->queueUrl);
     sendMessageRequest.SetMessageBody(message);
- 
+
     Aws::SQS::Model::SendMessageOutcome sendMessageOutcome = this->sqsClient->SendMessage(sendMessageRequest);
-     
+
     if(!sendMessageOutcome.IsSuccess() || sendMessageOutcome.GetResult().GetMessageId().length() == 0 )
       {
-	cout << "Error occurred during the sending " << endl;
-	ref.code=-1;
-	ref.body="Error occurred during message sending";
+        cout << "Error occurred during the sending " << endl;
+        ref.code=-1;
+        ref.body="Error occurred during message sending";
       }
-    else 
+    else
       {
-	ref.code=3;
-	ref.body=convertAwsStringToCharPtr(sendMessageOutcome.GetResult().GetMessageId());
-      }	
-     
+        ref.code=3;
+        ref.body=sendMessageOutcome.GetResult().GetMessageId().c_str();
```

Comment:
Slightly better would be to remove the c_str() since it is assigning to a std::string (and would make it clearer the member was a string).